src/utils/config.js

```diff
@@ -1,6 +1,7 @@
 import axios from "axios";
 
 export const STAGING_URL = "https://ah-codeofduty-staging.herokuapp.com/api";
+export const FE_URL= "https://ah-codeofduty-frontend-staging.herokuapp.com/";
```

Comment:
Good job on the feature, but these two URLs look quite similar, why define them twice